packages/config/config/devices/0x015f/mh-p220_3.2-255.255

```diff
@@ -0,0 +1,280 @@
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH-S220",
+	"description": "Micro Switch",
+	"devices": [
+		{
+			"productType": "0x2201",
+			"productId": "0x1252"
+		},
+		{
+			"productType": "0x2201",
+			"productId": "0x7202"
+		}
+	],
+	"firmwareVersion": {
+		"min": "3.2",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Switch 1",
+			"maxNodes": 5
+		},
+		"2": {
+			"label": "Switch 2",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"$import": "~/templates/master_template.json#state_after_power_failure_off_prev"
+		},
+		{
+			"#": "2",
+			"label": "External Switch Type",
+			"valueSize": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Momentary button",
+					"value": 0
+				},
+				{
+					"label": "Toggle (2-state) switch",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "2-State switch behavior",
+			"valueSize": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Toggle without specified on/off position",
+					"value": 0
+				},
+				{
+					"label": "Toggle with specified on/off position",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "4",
+			"label": "Off-timer",
```

Comment:
I don't understand how this label and the options fit together. Maybe there's a better label?

Reply:
Also, params 4 and 5 here look like they are an almost exact match to the old params 5 and 6?